### PR TITLE
chore(RHTAPWATCH-361): simple Splunk test

### PR DESCRIPTION
**Pull request by Omeramsc**
A test on a Splunk query to demonstrate GO test-framework. It's running the fake cluster and checks that the count of logs indexed for the test_index is as we expect.

depends on #20

### CODE REVIEW

File: splunk/splunk.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/redhat-appstudio/segment-bridge.git/containerfixture"
+)
+
+func getRecordsCount(host, port, owner, app, index string) (string, error) {
+	login := fmt.Sprintf("%s:%s", containerfixture.Username, containerfixture.Password)
+	endpoint := fmt.Sprintf(
+		"https://%s:%s/servicesNS/%s/%s/search/v2/jobs/export?output_mode=csv",
+		host, port, owner, app,
+	)
+	searchQuery := "search=" + (fmt.Sprintf("search index=%s | stats count", index))
```
Comment (yftacherzog) [inline splunk/splunk.go:18]: Do we need the outer parentheses here?

Comment (Omeramsc) [reply splunk/splunk.go:18]: Nope, good catch.

**Review by yftacherzog — APPROVED** [review]

**Review by ifireball — CHANGES_REQUESTED** [review]
Not looking at the code here yet, I think this can be a part of #20, so that well have some tests included for the code there.

**Comment by Omeramsc** [comment]
@b

> Not looking at the code here yet, I think this can be a part of #20, so that well have some tests included for the code there.

I get the value of having the tests with the framework, but I separated them on purpose. #20 is big enough and other changes depends on it, adding GO test code (which may need multiple changes) will prolong it even further.

**Comment by ifireball** [comment]
> > Not looking at the code here yet, I think this can be a part of #20, so that well have some tests included for the code there.
> 
> I get the value of having the tests with the framework, but I separated them on purpose. #20 is big enough and other changes depends on it, adding GO test code (which may need multiple changes) will prolong it even further.

Are we cheating ourselves? You can't reliably merge that code without having this code - there are both parts of the same work - if you really need so separation use different commits within the same PR.

**Pull request closed without merging**